intro/HarryKinigopoulos.md

```diff
@@ -0,0 +1,15 @@
+<p align="center">
+<img src="https://www.instagram.com/p/CCqxB7DAacp/" height="400">
+
+<h1 align="center">[Charalampos Kynigopoulos]</h1>
+
+<h2 align="center">[Computer Science Student]</h2>
```

Comment:
The template `[]` meant it was variable value. Please remove them from your file.